mypy/main.py

```diff
@@ -256,6 +256,9 @@ def add_invertible_flag(flag: str,
                         version='%(prog)s ' + __version__)
     parser.add_argument('--python-version', type=parse_version, metavar='x.y',
                         help='use Python x.y')
+    parser.add_argument('--python', action='store',
```

Comment:
I am making the name change, but as for a default based on `options.python_version`, I'm afraid I don't understand what you mean? Are you suggesting that if I say `--python-version=3.4` mypy should make a guess about where python 3.4 is installed? I'm afraid that may not be the most straight forward (I'm not aware of a means of finding where a python executable resides in a cross platform manner). As a possible work-around/compromise, perhaps `python-executable` could be added as an option to mypy.ini, so that one needn't keep entering it.

Reply:
On Unix one would use `which python3.4` -- this searches `$PATH` (in fact you don't need to literally call `which`, you can just look at `os.getenv('PATH')` and see if there's an entry containing `python3.4`). If that doesn't give anything I'd try `which python3` and run it with `-V` to find out what minor version it is.